frc_msgs/msg/RobotState.msg

```diff
@@ -0,0 +1,26 @@
+# Corresponds to frc::RobotController
+Header header
+int32 fpga_version
+int64 fpga_revision
```

Comment:
Why is version 32 and revision 64? A lil overkill?

Reply:
This was done to match the return type from the corresponding wpilib function.

From `wpilibc/frc/RobotController.h` @ `static int64_t GetFPGARevision()`:

```
The format of the revision is 3 numbers. The 12 most significant bits are
the Major Revision. The next 8 bits are the Minor Revision. The 12 least
significant bits are the Build Number.
```

Frankly not sure why it wasn't a uint64 -.- but whatever. We'll match them.